requirements.txt

```diff
@@ -1,23 +1,30 @@
+appdirs==1.4.3
+attrs==19.3.0
 beautifulsoup4==4.9.0
-soupsieve==1.9.5
-Flask==1.1.2
-Jinja2==2.11.1
-MarkupSafe==1.1.1
-Werkzeug==1.0.1
-click==7.1.1
-itsdangerous==1.1.0
+black==19.10b0
 certifi==2020.4.5.1
 chardet==3.0.4
+chromedriver-installer==0.0.6
```

Comment:
I think the flask installation has to come before the chromedriver installation. Pulled this branch into a fresh repository with a fresh venv and got an error during the chromedriver installation "No module named flask" and ./run.sh failed.

Ran ./run.sh a second time immediately after (with the same venv) and everything went fine (app ran, no errors during package installations).

Reply:
This is interesting. I think your idea for testing various use-cases (i.e. already set up env vs new user) is spot on. Thanks for thinking of the multiple scenarios. I apprecaite it!

Reply:
Package installation went smoothly